Modules/Filtering/ImageGrid/test/itkBSplineControlPointImageFunctionTest.cxx

```diff
@@ -71,9 +71,40 @@ itkBSplineControlPointImageFunctionTest(int, char *[])
   size.Fill(101);
 
   bspliner->SetOrigin(origin);
+  ITK_TEST_SET_GET_VALUE(origin, bspliner->GetOrigin());
+
   bspliner->SetSpacing(spacing);
+  ITK_TEST_SET_GET_VALUE(spacing, bspliner->GetSpacing());
+
   bspliner->SetSize(size);
-  bspliner->SetSplineOrder(3);
+  ITK_TEST_SET_GET_VALUE(size, bspliner->GetSize());
+
+  unsigned int bSplineOrderValue = 3;
+  bspliner->SetSplineOrder(bSplineOrderValue);
+  for (auto i : bspliner->GetSplineOrder())
+  {
+    std::cerr << "Test failed!" << std::endl;
```

Comment:
`if (i!=bSplineOrderValue)`?

Reply:
Test failed!
Error in itk::BSplineControlPointImageFunction::GetSplineOrder
Expected: 3, but got: 3

Reply:
Yep, my bad. Thanks for the pointer @dzenanz . Addressed in 74905a2.